ansible/roles/jenkins-worker/vars/main.yml

```diff
@@ -92,6 +92,7 @@ java_path: {
   'macos10.14': 'java',
   'macos10.15': 'java',
   'macos10.16': 'java',
+  'macos11': 'java',
```

Comment:
@AshCripps You'll probably need a `macos11` equivalent of https://github.com/nodejs/build/blob/664fabdb7b6e6990d78bffe450b5fad2dfe2421e/ansible/roles/java-base/vars/main.yml#L16 as well?

Reply:
yep thats what Ive got locally will push up will all working

Reply:
we do - I added checks for level and arch now but may need to rethink this part - although it may be a none issue with adopts arm builds being available. also will need to tweak the ci labels